client/src/components/main.js

```diff
@@ -12,19 +12,34 @@ class Main extends Component {
       str: ""
     };
     this.url = "/getCards";
+    // this.updateDisplay = this.updateDisplay.bind(this);
   }
 
   componentDidMount() {
     fetch(this.url)
-      .then(d => d.json())
+      .then(res => res.json())
       .then(d => {
-        this.setState({ cards: d });
+        this.setState({
+          cards: d.data.map(card => {
+            return { ...card, active: false };
+          })
+        });
       });
   }
 
+  updateDisplay = id => {
+    this.state.cards.map(c => {
+      if (c._id === id) {
+        c.active = !c.active;
+      } else c.active = false;
+    });
+    this.setState({ state: this.state });
```

Comment:
What's going on here?

Reply:
toggle the display of the selected card, then  force re-render the component